libs/common-system/src/main/java/org/codice/ddf/configuration/SystemInfo.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration;
+
+public class SystemInfo {
+
+    public static final String SITE_NAME = "org.codice.ddf.system.siteName";
+
+    public static final String SITE_CONTATCT = "org.codice.ddf.system.siteContact";
+
+    public static final String VERSION = "org.codice.ddf.system.version";
+
+    public static final String ORGANIZATION = "org.codice.ddf.system.organization";
+
+    public SystemInfo() {
+
+    }
+
+    public String getSiteName() {
```

Comment:
It would be nice to return an empty string instead of a null if there is no value.